### PR TITLE
Minor: Grammar fixes. Remove X

### PR DESCRIPTION
**Pull request by estelle**
I went in to remove reference to a toxic social media platform. 
Fixed some grammar as well.
Merged in a second PR that removes X, so a few more removals

### CODE REVIEW

File: files/en-us/learn/forms/form_validation/index.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ You can constrain the character length of all text fields created by {{HTMLEleme
 A field is invalid if it has a value and that value has fewer characters than the [`minlength`](/en-US/docs/Web/HTML/Attributes/minlength) value or more than the [`maxlength`](/en-US/docs/Web/HTML/Attributes/maxlength) value.
 
 Browsers often don't let the user type a longer value than expected into text fields. A better user experience than just using `maxlength` is to also provide character count feedback in an accessible manner and let them edit their content down to size.
-An example of this is the character limit seen on Twitter when Tweeting. JavaScript, including [solutions using `maxlength`](https://github.com/mimo84/bootstrap-maxlength), can be used to provide this.
+An example of this is the character limit seen on Mastodon when Tooting. JavaScript, including [solutions using `maxlength`](https://github.com/mimo84/bootstrap-maxlength), can be used to provide this.
```
Comment (Josh-Cena) [inline files/en-us/learn/forms/form_validation/index.md:245]:
```suggestion
An example of this is the character limit when posting on social media. JavaScript, including [solutions using `maxlength`](https://github.com/mimo84/bootstrap-maxlength), can be used to provide this.
```

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 2]
```diff
@@ -182,12 +182,6 @@ One effect of this is that when you link to MDN Web Docs on Facebook, the link a
 
 ![Open graph protocol data from the MDN homepage as displayed on facebook, showing an image, title, and description.](facebook-output.png)
 
-Twitter also has its own similar proprietary metadata called [Twitter Cards](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards), which has a similar effect when the site's URL is displayed on twitter.com. For example:
```
Comment (Josh-Cena) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:185]: This is still important as Twitter cards are also read by other tools AFAIK, and many developers still need to support it. It's not in the scope of MDN to boycott any technology.

File: files/en-us/learn/javascript/client-side_web_apis/index.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Basic knowledge of [HTML](/en-US/docs/Learn/HTML) and [CSS](/en-US/docs/Learn/CS
...
 - [Third party APIs](/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Third_party_APIs)
-  - : The APIs we've covered so far are built into the browser, but not all APIs are. Many large websites and services such as Google Maps, Twitter, Facebook, PayPal, etc. provide APIs allowing developers to make use of their data (e.g. displaying your twitter stream on your blog) or services (e.g. displaying custom Google Maps on your site, or using Facebook login to log in your users). This article looks at the difference between browser APIs and 3rd party APIs and shows some typical uses of the latter.
+  - : The APIs we've covered so far are built into the browser, but not all APIs are. Many large websites and services such as Google Maps, Mastodon, Facebook, PayPal, etc. provide APIs allowing developers to make use of their data (e.g. displaying your Mastodon stream on your blog) or services (e.g. displaying custom Google Maps on your site, or using Facebook login to log in your users). This article looks at the difference between browser APIs and 3rd party APIs and shows some typical uses of the latter.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/client-side_web_apis/index.md:37]:
```suggestion
  - : The APIs we've covered so far are built into the browser, but not all APIs are. Many large websites and services such as Google Maps, Facebook, PayPal, etc. provide APIs allowing developers to make use of their data or services (e.g. displaying custom Google Maps on your site, or using Facebook login to log in your users). This article looks at the difference between browser APIs and 3rd party APIs and shows some typical uses of the latter.
```

Mastodon is not a "large website or service" and is going to be more confusing than helpful.

**Review by Josh-Cena — COMMENTED** [review]
Overall I like to decouple us from proprietary products but sometimes it goes too far

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:185]: The open graph protocol, while run by Facebook, is open source. Twitter is proprietary, private, and closed. And I am not certain anymore about their privacy and security. We should be favoring alternative, and definitely not be including tech we don't trust on MDN.

**Review by chrisdavidmills — APPROVED** [review]
FWIW, I think these changes are fine. We still provide suitable examples, the name drops that were removed are unnecessary, and the Twitter API in its current form is arguably not very beginner-friendly (or free), so is not super helpful these days.

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**